src/main/java/com/github/paweladamski/httpclientmock/HttpClientMockBuilder.java

```diff
@@ -110,6 +138,17 @@ public HttpClientMockBuilder withBody(Matcher<String> matcher) {
     return this;
   }
 
+  /**
+   * Adds body condition. Request body must contain the given URL-encoded form parameters (typically used in POST requests). Alternatively, parameters may be specified individually using {@link #withFormParameter(String, Matcher)}.
+   * 
+   * @param parameters the parameters
+   * @return condition builder
+   */
+  public HttpClientMockBuilder withBody(MatchersMap<String, String> parameters) {
```

Comment:
I thought you changed it to `withFormParameters'

Reply:
Fixed in cb31d06f00a64792e996a5080e783986c22bbb16.